src/executor/admin/SubmitJobExecutor.cpp

```diff
@@ -18,16 +18,25 @@ folly::Future<Status> SubmitJobExecutor::execute() {
 
     auto *sjNode = asNode<SubmitJob>(node());
     auto jobOp = sjNode->jobOp();
-    meta::cpp2::AdminCmd cmd = meta::cpp2::AdminCmd::COMPACT;
+    meta::cpp2::AdminCmd cmd;
     if (jobOp == meta::cpp2::AdminJobOp::ADD) {
         std::vector<std::string> params;
         folly::split(" ", sjNode->params().front(), params, true);
```

Comment:
Only the first element of sjNode->params() is taken here, and the other elements are lost.